File: erettsegi_emelt/2020_may/java/IdojarasAdat.java
import java.time.*;

public class IdojarasAdat {
        
    public final String telepules;
    public final LocalTime ido;
    public final String szelIrany;
    public final int szelErosseg;
    public final int homerseklet;
    
    public IdojarasAdat(String line) {
        var split = line.split(" ");
        var teljesIdo = Integer.parseInt(split[1]);
        
        telepules = split[0];
        ido = LocalTime.of(teljesIdo / 100, teljesIdo % 100);
        szelIrany = split[2].substring(0, 3);
        szelErosseg = Integer.parseInt(split[2].substring(3));
        homerseklet = Integer.parseInt(split[3]);
    }
}

File: erettsegi/emelt/2020_may/c#/IdojarasAdat.cs
using System;

public class IdojarasAdat {
        
    public readonly string telepules;
    public readonly DateTime ido;
    public readonly string szelIrany;
    public readonly int szelErosseg;
    public readonly int homerseklet;
    
    public IdojarasAdat(String line) {
        var split = line.Split(' ');
        var teljesIdo = int.Parse(split[1]);
        var now = DateTime.Now;

        telepules = split[0];
        ido = new DateTime(now.Year, now.Month, now.Day, teljesIdo / 100, teljesIdo % 100, 0);
        szelIrany = split[2].Substring(0, 3);
        szelErosseg = int.Parse(split[2].Substring(3));
        homerseklet = int.Parse(split[3]);
    }
}